src/components/access_group/AccessGroupMembers.vue

```diff
@@ -0,0 +1,327 @@
+<template>
```

Comment:
I'll do a summary for this file:
- Moving to a single list for accessibility.
- In general this component should be more driven by the API. Pagination, filtering and searching is part of the API and right now searching and filtering only works for the already fetched members. To display more then 20 members we need to fetch additional members like the top level search does.
- Deleting a search (emptying the search field and hitting enter does not work).
- The tabs containers are not keyboard select-able (and do not survive a reload (not that important))

Reply:
Good call. This has been adjusted!